softwareComponents/configuration/include/configuration/rofiworld.hpp

```diff
@@ -445,9 +445,14 @@ class SimpleCollision {
     bool operator()( const Module& a, const Module& b, Matrix posA, Matrix posB ) {
         for ( auto pA : a.getOccupiedRelativePositions() ) {
             for ( auto pB : b.getOccupiedRelativePositions() ) {
-                if ( rofi::configuration::matrices::equals( static_cast< Matrix >( posA * pA ), posB * pB ) ) {
+
+                Vector A = static_cast< Matrix >( posA * pA ) * Vector{ 0, 0, 0, 1 };
+                Vector B = static_cast< Matrix >( posB * pB ) * Vector{ 0, 0, 0, 1 };
+                auto magnitude = []( Vector vec ){
+                    return std::sqrt( vec[ 0 ] * vec[ 0 ] + vec[ 1 ] * vec[ 1 ] + vec[ 2 ] * vec[ 2 ] );
+                };
+                if ( magnitude( A - B ) < 0.99 )
```

Comment:
Proč používáš `0.99`? je to nějaká arbitrary value, která není nikde dokumentovaná.
Jestli chceš mít precision, dej to jako argument (Pravděpodobně jako argument konstruktoru.)

Reply:
Původně tam byla jednička (vzdálenost mezi dvěma středy v mřížce), ale to způsobovalo problémy v testech kvůli nepřesnosti (1 < 1); po použití funkcí z `matrices` to ale vypadá, že to funguje i bez tohohle magického čísla.

Reply:
Kdyžtak moje výtka nebyla na použití menšího čísla na porovnání, ale že to magické číslo nemusí být magické, když ho pojmenuješ - když dáš e.g. precision=0.01, tak to přestane být magické

Reply:
Já si jen dovolím doplnit, že tady bych zrovna chtěl pojmenovat **a** vysvětlit v komentáři. Ano, 0.99 je tady OK. Skoro stejně tak jako 0.8. Ale je potřeba říci i proč.